code/modules/lighting/lighting_source.dm

```diff
@@ -140,23 +151,23 @@
 // This exists so we can cache the vars used in this macro, and save MASSIVE time :)
 // Most of this is saving off datum var accesses, tho some of it does actually cache computation
 // You will NEED to call this before you call APPLY_CORNER
-#define SETUP_CORNERS_CACHE(lighting_source)                               \
-	var/_turf_x = lighting_source.pixel_turf.x;                            \
-	var/_turf_y = lighting_source.pixel_turf.y;                            \
-	var/_turf_z = lighting_source.pixel_turf.z;                            \
-	var/list/_sheet = get_sheet();                                         \
-	var/list/_multiz_sheet = list();                                       \
-	if(!!GET_LOWEST_STACK_OFFSET(source_turf.z)) {                         \
-		_multiz_sheet = get_sheet(multiz = TRUE);                          \
-	}                                                                      \
-	var/_range_offset = CEILING(lighting_source.light_range, 1) + 0.5 + 2; \
-	var/_multiz_offset = SSmapping.max_plane_offset + 1;                   \
-	var/_light_power = lighting_source.light_power;                        \
-	var/_applied_lum_r = lighting_source.applied_lum_r;                    \
-	var/_applied_lum_g = lighting_source.applied_lum_g;                    \
-	var/_applied_lum_b = lighting_source.applied_lum_b;                    \
-	var/_lum_r = lighting_source.lum_r;                                    \
-	var/_lum_g = lighting_source.lum_g;                                    \
+#define SETUP_CORNERS_CACHE(lighting_source)                                                               \
```

Comment:
golly this is a wacky diff that's still mal-aligned, doesn't have to be fixed now but it'd look a heck of a lot better with just one space after each line rather than exacerbating the issue with more tabs. just FYIing for future improvement

Reply:
the idea is to hide the \n part of it as much as we can, and have it line up nice.
it's how pjb did macros, and it's just carried over from then

Reply:
yeah but it doesn't line up nice